### PR TITLE
[CosmosDB] Introduce new features from latest swagger 11-15-Preview and add test coverage

### PR DESCRIPTION
**Pull request by ddhamotharan**
# Contributing to the Azure SDK

Introducing the below mentioned new features of CosmosDB from latest swagger 11-15-Preview and add test coverage

- InAccount Restore of Deleted Tables and Gremlin Database and Graph resources
- Cross region restore
- Burst Capacity

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by ddhamotharan** [comment]
@live1206 We've fixed the issue with session records due to new test. Can you please review the PR now?
Thank you

**Comment by rsharm7** [comment]
@live1206 Could you please merge the PR?

File: sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/BackupResource.cs [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+namespace Azure.ResourceManager.CosmosDB
+{
+    /// <summary>
+    /// A Class representing a BackupResource along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier" /> you can construct a <see cref="BackupResource" />
+    /// from an instance of <see cref="ArmClient" /> using the GetBackupResource method.
+    /// Otherwise you can get one from its parent resource <see cref="CassandraClusterResource" /> using the GetBackupResource method.
+    /// </summary>
+    public partial class BackupResource : ArmResource
```
Comment (live1206) [inline sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/BackupResource.cs:25]: `BackupResource` is too generic, consider renaming it to `CassandraClusterBackupResource`.

**Review by live1206 — APPROVED** [review]

**Pull request merged**